.gitignore

```diff
@@ -0,0 +1,13 @@
+*.swp
+builds
+build
+.impconfig
+.imptest
```

Comment:
.imptest should not be excluded

Reply:
I excluded it because it has the API key in it.

Reply:
Oh I see. I set in the environment variable. It doesn't have to be a part of the config.